extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -381,15 +388,50 @@ export abstract class PostgresParametersPage extends DashboardPage {
 			return true */
 			this.saveButton!.enabled = true;
 			this.discardButton!.enabled = true;
+			this.collectChangedComponents(name);
 			return true;
 		}
 	}
 
 	protected createParameterComponents(engineSetting: EngineSettingsModel): ParametersModel {
 
-		// Container to hold input component and information bubble
-		const valueContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		// Can reset individual parameter
+		const resetParameterButton = this.modelView.modelBuilder.button().withProps({
+			iconPath: IconPathHelper.reset,
+			title: loc.resetToDefault,
+			width: '20px',
+			height: '20px',
+			enabled: true
+		}).component();
 
+		this.disposables.push(
+			resetParameterButton.onDidClick(async () => {
+				try {
+					await vscode.window.withProgress(
+						{
+							location: vscode.ProgressLocation.Notification,
+							title: loc.updatingInstance(this._postgresModel.info.name),
+							cancellable: false
+						},
+						async (_progress, _token): Promise<void> => {
+							const session = await this._postgresModel.controllerModel.acquireAzdataSession();
+							try {
+								await this.resetParameter(engineSetting.parameterName!, session);
+							} finally {
+								session.dispose();
+							}
+							await this._postgresModel.refresh();
```

Comment:
Maybe have this in a separate try/catch - we should still show the "instance updated" message even if the refresh failed (but want to still display an error about the refresh failing)

Reply:
I also realized this is a problem across other pages as well, so if you haven't done so already we can just fix those all up at once instead. Whatever works for you.

Reply:
will do in separate PR :-)